package.json

```diff
@@ -43,6 +43,7 @@
     "prettier": "^1.13.5",
     "raf": "^3.4.0",
     "raw-loader": "^0.5.1",
+    "react-redux-test-utils": "^0.1.1",
```

Comment:
this should not be needed for packaging, hence does not require a packaging PR. /cc @ekohl